packages/salesforcedx-vscode-lightning/src/index.ts

```diff
@@ -89,16 +89,9 @@ export async function activate(extensionContext: ExtensionContext) {
   // Start the Aura Language Server
 
   // Setup the language server
+  const serverPath = extensionContext.extension.packageJSON.serverPath;
```

Comment:
This reads the value from the package.json and uses it to specify the path the server file.  This is what allows us to have different locations between local dev and vsix.